packages/office-ui-fabric-react/src/components/DetailsList/DetailsList.tsx

```diff
@@ -246,11 +255,7 @@ export class DetailsList extends BaseComponent<IDetailsListProps, IDetailsListSt
       selectAllVisibility = SelectAllVisibility.none;
     }
 
-    const {
-      onRenderDetailsHeader = this._onRenderDetailsHeader
-    } = this.props;
-
-    const DetailsHeader = (props: IDetailsHeaderProps) => onRenderDetailsHeader(props, this._onRenderDetailsHeader);
+    const DetailsHeader = this._DetailsHeader;
```

Comment:
I'm trying to take advantage of `<Function>` syntax to keep the markup below clean. For this to work I need to store that function as a field. We already have `_onRenderDetailsHeader` as its own member, so I need to give the final function a new name.